connection_plugins/machinectl.py

```diff
@@ -38,7 +38,7 @@
 from ansible.errors import AnsibleError
 from ansible.plugins.connection import ConnectionBase
 from ansible.utils.vars import merge_hash
-from ansible.utils.unicode import to_bytes
+from ansible.module_utils._text import to_bytes
```

Comment:
_text means a private module, right? So maybe we shouldn't use it, but ah well. Open to ideas